gms/feature_service.hh

```diff
@@ -26,17 +26,12 @@ namespace gms {
 
 class feature_service;
 
-struct feature_config {
-private:
-    std::set<sstring> _disabled_features;
-    feature_config();
-
-    friend class feature_service;
-    friend feature_config feature_config_from_db_config(db::config& cfg, std::set<sstring> disabled);
+// Disable some features additionally to the ones disabled by `db::config`.
+// Use for tests only!
```

Comment:
I didn't understand why we need this. Can't we already disable features manually via the configuration? Why do tests need a second mechanism?

Reply:
This `disabled` argument was introduced long ago to facilitate this:
```
test/boost/truncation_migration_test.cc:36:    cfg.disabled_features = { "TRUNCATION_TABLE" };
```
now this part is gone, but ...

Reply:
upd: the quoted code was prior to bb3a71529a93deb18679916a55e3924b7f37f181